apps/dashboard-for-dapps/src/routes/index.tsx

```diff
@@ -334,15 +216,17 @@ function CredentialDetails({
                           _hover={{ transform: "scale(1.02)" }}
                           onClick={() => openImageInNewTab(value)}
                         >
-                          <Image
-                            src={value}
-                            alt="Identification document front"
-                            rounded="md"
-                            loading="lazy"
-                            width="120px"
-                            height="120px"
-                            title="Click to open the image in full size"
-                          />
+                          <chakra.button className="button">
+                            <Image
+                              src={value}
+                              alt="Identification document front"
```

Comment:
If we're hardcoding alternate text, we should probably make it more generic no? Like "Image from credential"